bindings/experimental/distrdf/python/DistRDF/Ranges.py

```diff
@@ -282,7 +285,7 @@ def get_balanced_ranges(nentries, npartitions):
     return ranges
 
 
-def get_clustered_ranges(clustersinfiles, npartitions, treename, friend_info):
+def get_clustered_ranges(clustersinfiles, npartitions, friend_info):
```

Comment:
`treename` is removed because it was not actually used, right? It should be removed from the docs below too.

Reply:
Mmm indeed it was not actually used because currently we get the treename as a variable in the mapper from the outside `BaseBackend.execute` function. If we get to a mapper per head node function design, then we would probably again include the treename in the range object